--- src/texturegenerator.h.orig
+++ src/texturegenerator.h
@@ -4,6 +4,7 @@
 #include <vector>
 #include <QImage>
 #include <QColor>
+#include <QPainterPath>
 #include <QPixmap>
 #include "outcome.h"
 #include "model.h"
--- src/contourtopartconverter.cpp.orig
+++ src/contourtopartconverter.cpp
@@ -1,5 +1,6 @@
 #include <QImage>
 #include <QPainter>
+#include <QPainterPath>
 #include <cmath>
 #include <QUuid>
 #include <cmath>
