--- src/extension/internal/pdfinput/pdf-input.cpp.orig
+++ src/extension/internal/pdfinput/pdf-input.cpp
@@ -329,7 +329,7 @@
     _render_thumb = true;
 
     // Create PopplerDocument
-    Glib::ustring filename = _pdf_doc->getFileName()->getCString();
+    Glib::ustring filename = _pdf_doc->getFileName()->c_str();
     if (!Glib::path_is_absolute(filename)) {
         filename = Glib::build_filename(Glib::get_current_dir(),filename);
     }
--- src/extension/internal/pdfinput/pdf-parser.cpp.orig
+++ src/extension/internal/pdfinput/pdf-parser.cpp
@@ -2201,8 +2201,8 @@
   }
   if (printCommands) {
     printf("  font: tag=%s name='%s' %g\n",
-	   font->getTag()->getCString(),
-	   font->getName() ? font->getName()->getCString() : "???",
+	   font->getTag()->c_str(),
+	   font->getName() ? font->getName()->c_str() : "???",
 	   args[1].getNum());
     fflush(stdout);
   }
@@ -2453,7 +2453,7 @@
     double lineX = state->getLineX();
     double lineY = state->getLineY();
     oldParser = parser;
-    p = s->getCString();
+    p = s->c_str();
     len = s->getLength();
     while (len > 0) {
       n = font->getNextChar(p, len, &code,
@@ -2502,7 +2502,7 @@
 
   } else {
     state->textTransformDelta(0, state->getRise(), &riseX, &riseY);
-    p = s->getCString();
+    p = s->c_str();
     len = s->getLength();
     while (len > 0) {
       n = font->getNextChar(p, len, &code,
--- src/extension/internal/pdfinput/svg-builder.cpp.orig
+++ src/extension/internal/pdfinput/svg-builder.cpp
@@ -1032,7 +1032,7 @@
     GfxFont *font = state->getFont();
     // Store original name
     if (font->getName()) {
-        _font_specification = font->getName()->getCString();
+        _font_specification = font->getName()->c_str();
     } else {
         _font_specification = "Arial";
     }
@@ -1059,7 +1059,7 @@
 
     // Font family
     if (font->getFamily()) { // if font family is explicitly given use it.
-        sp_repr_css_set_property(_font_style, "font-family", font->getFamily()->getCString());
+        sp_repr_css_set_property(_font_style, "font-family", font->getFamily()->c_str());
     } else { 
         int attr_value = 1;
         sp_repr_get_int(_preferences, "localFonts", &attr_value);
