--- Makefile.orig
+++ Makefile
@@ -24,7 +24,7 @@
 PREFIX?=	/usr/local
 X11BASE?=	/usr/X11R6
 
-PKGLIBS=	x11 xft xext xpm gtk+-2.0
+PKGLIBS=	x11 xft xext xpm gtk+-2.0 freetype2
 
 CC?=		cc
 CFLAGS+=	-O2 -Wall -Wunused \
@@ -80,8 +80,8 @@
 	$(CC) -o $@ $(AEMENU_OBJ) $(LDFLAGS)
 
 install: all
-	mkdir -p $(BINDIR) $(MANDIR)
-	install -s $(BIN) $(BINDIR)
+	mkdir -p $(DESTDIR)$(BINDIR) $(DESTDIR)$(MANDIR)
+	install -s $(BIN) $(DESTDIR)$(BINDIR)
 
 clean:
 	rm -f $(BIN) $(OBJ)
